massa-final-state/src/final_state.rs

```diff
@@ -821,13 +651,233 @@ impl FinalState {
         Ok(())
     }
 
-    /// Initialize the execution trail hash to zero.
-    pub fn init_execution_trail_hash_to_batch(&mut self, batch: &mut DBBatch) {
+    /// Initializes a `FinalState` from a snapshot. Currently, we do not use the final_state from the ledger,
+    /// we just create a new one. This will be changed in the follow-up.
```

Comment:
Realized that this part is outdated, can you remove it real quick? :)

Reply:
DOne